packages/contentstack-branch/src/branch/merge-handler.ts

```diff
@@ -199,15 +203,36 @@ export default class MergeHandler {
 
   async executeMerge(mergePayload) {
     let spinner;
+    let scriptFolderPath;
     try {
       if (!this.mergeSettings.mergeComment) {
         this.mergeSettings.mergeComment = await askMergeComment();
         mergePayload.merge_comment = this.mergeSettings.mergeComment;
       }
+
+      if (enableEntryExp) {
+        scriptFolderPath = generateMergeScripts(this.mergeSettings.mergeContent);
+      }
+
+      cliux.loader('Merging the changes');
       spinner = cliux.loaderV2('Merging the changes...');
       const mergeResponse = await executeMerge(this.stackAPIKey, mergePayload);
       cliux.loaderV2('', spinner);
       cliux.success(`Merged the changes successfully. Merge UID: ${mergeResponse.uid}`);
+
+      if (enableEntryExp) {
+        let newScriptFolderPath = renameScriptFolder(mergeResponse.uid, scriptFolderPath);
+
+        cliux.success(`Success! Merge has been completed and we have generated entry migration files in folder
```

Comment:
Just message entry file information, since we already printed the success message merge beforer